src/components/paragraph/stanford-lists.tsx

```diff
@@ -93,6 +83,16 @@ const ListParagraph = async ({paragraph}: Props) => {
           totalItems={addLoadMore ? totalItems : viewItems.length}
         />
       )}
+
+      {paragraph.suListButton?.url && (
+        <DrupalLinkButton
+          className="mx-auto mt-0"
+          href={paragraph.suListButton.url}
+          {...paragraph.suListButton.attributes}
```

Comment:
if `paragraph.suListButton.attributes` contains a className prop, it will override your className here. I'd suggest either applying that class from above or using `twMerge`
```suggestion
          {...paragraph.suListButton.attributes}
          className={twMerge("mx-auto mt-0", paragraph.suListButton.attributes?.className)}
          href={paragraph.suListButton.url}
```